.circleci/config.yml

```diff
@@ -70,11 +70,15 @@ workflows:
       - build_and_test:
           filters:
             branches:
-              ignore: /trunk/
+              ignore:
+                - /trunk/
+                - 8.x
       - publish:
           filters:
             branches:
-              only: /trunk/
+              only:
+                - /trunk/
+                - 8.x
```

Comment:
8.x is still a factor?

Reply:
Well, today it is. And it will be until I get 10.x done. We can probably delete this branch later once everything is working great with 10. I just need a release with 9 while still providing a path to update or patch 8.x while it's in the works.